homeassistant/components/delijn/sensor.py

```diff
@@ -0,0 +1,123 @@
+"""Support for De Lijn (Flemish public transport) information."""
+import logging
+
+from pydelijn.api import Passages
+import voluptuous as vol
+
+from homeassistant.components.sensor import PLATFORM_SCHEMA
+from homeassistant.const import (
+    ATTR_ATTRIBUTION, DEVICE_CLASS_TIMESTAMP)
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
+import homeassistant.helpers.config_validation as cv
+from homeassistant.helpers.entity import Entity
+
+_LOGGER = logging.getLogger(__name__)
+
+ATTRIBUTION = "Data provided by data.delijn.be"
+
+CONF_NEXT_DEPARTURE = 'next_departure'
+CONF_STOP_ID = 'stop_id'
+CONF_API_KEY = 'api_key'
+CONF_NUMBER_OF_DEPARTURES = 'number_of_departures'
+
+DEFAULT_NAME = 'De Lijn'
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
+    vol.Required(CONF_API_KEY): cv.string,
+    vol.Required(CONF_NEXT_DEPARTURE): [{
+        vol.Required(CONF_STOP_ID): cv.string,
+        vol.Optional(CONF_NUMBER_OF_DEPARTURES, default=5): cv.positive_int}]
+})
+
+
+async def async_setup_platform(
+        hass, config, async_add_entities, discovery_info=None):
+    """Create the sensor."""
+    api_key = config[CONF_API_KEY]
+    name = DEFAULT_NAME
+
+    session = async_get_clientsession(hass)
+
+    sensors = []
+    for nextpassage in config[CONF_NEXT_DEPARTURE]:
+        stop_id = nextpassage[CONF_STOP_ID]
+        number_of_departures = nextpassage[CONF_NUMBER_OF_DEPARTURES]
+        line = Passages(hass.loop,
+                        stop_id,
+                        number_of_departures,
+                        api_key,
+                        session,
+                        True)
+        await line.get_passages()
+        if line.passages is None:
+            _LOGGER.warning("No data recieved from De Lijn")
+            return
+        sensors.append(DeLijnPublicTransportSensor(line, name))
+
+    async_add_entities(sensors, True)
+
+
+class DeLijnPublicTransportSensor(Entity):
+    """Representation of a Ruter sensor."""
+
+    def __init__(self, line, name):
+        """Initialize the sensor."""
+        self.line = line
+        self._attributes = {}
+        self._name = name
+        self._state = None
+        self._attributes[ATTR_ATTRIBUTION] = ATTRIBUTION
```

Comment:
Include the attribution when creating the dict.

Reply:
Updated in the latest commit